src/web3Admin.js

```diff
@@ -93,6 +93,13 @@ module.exports = {
         	methods:
         	[
         	new web3._extend.Method({
+        		name: 'debugTrace',
```

Comment:
I guess it would be better to just call it `trace`, if the full method is called `debug_trace`